x-pack/filebeat/module/aws/vpcflow/config/input.yml

```diff
@@ -0,0 +1,134 @@
+{{ if eq .input "s3" }}
+
+type: s3
+queue_url: {{ .queue_url }}
+credential_profile_name: {{ .credential_profile_name }}
+
+{{ else if eq .input "file" }}
+
+type: log
+paths:
+  {{ range $i, $path := .paths }}
+  - {{$path}}
+    {{ end }}
+exclude_files: [".gz$"]
+
+{{ end }}
+
+processors:
+  - drop_event:
```

Comment:
Not sure why, `exclude_lines` works for generating local `*.log-expected.json` files but when running with ES for testing, it's still reading the log hearder lines. But this `drop_event` processor works for both scenarios.